packages/protoc-gen-connect-query-react/package.json

```diff
@@ -21,13 +21,16 @@
   "scripts": {
     "clean": "rm -rf dist",
     "build": "pnpm clean && tsc --project tsconfig.build.json",
-    "format": "prettier . --write --ignore-path ./.eslintignore && eslint . --fix && license-header",
+    "format": "prettier . --write --ignore-path ./.eslintignore && eslint . --fix && license-header && tsc --noEmit --project tsconfig.json",
```

Comment:
This change, plus the addition of the `ts_nocheck` flag sent to the buf plugin make it so we can run tsc to validate the resulting content.